src/test/java/eu/yals/test/ui/VaadinTest.java

```diff
@@ -30,155 +30,160 @@
-
-    super.setup();
-    Parameters.setScreenshotErrorDirectory(REPORT_DIRECTORY);
-  }
-
-  protected void open(String relativeOrAbsoluteUrl) {
-    final String PROTOCOL_MARKER = "://";
-    boolean isUrlAbsolute = relativeOrAbsoluteUrl.contains(PROTOCOL_MARKER);
-    if (isUrlAbsolute) {
-      getDriver().get(relativeOrAbsoluteUrl);
-    } else {
-      getDriver().get(BASE_URL + relativeOrAbsoluteUrl);
-    }
-  }
-
-  protected YalsElement $$(String cssSelector) {
-    return YalsElement.wrap(findElement(By.cssSelector(cssSelector)));
-  }
-
-  protected VaadinElement $$(TestBenchElement element) {
-    return VaadinElement.wrap(element);
-  }
-
-  @BrowserConfiguration
-  public List<DesiredCapabilities> getBrowserConfiguration() {
-    List<TestApp.Browser> testBrowsers = TestUtils.getTestBrowsers();
-    List<DesiredCapabilities> browsers = new ArrayList<>();
-
-    if (testBrowsers.contains(TestApp.Browser.CHROME)) {
-      ChromeOptions options = new ChromeOptions();
-      options.addArguments("enable-automation");
-      options.addArguments("--headless");
-      options.addArguments("--window-size=1920,1080");
-      options.addArguments("--no-sandbox");
-      options.addArguments("--disable-extensions");
-      options.addArguments("--dns-prefetch-disable");
-      options.addArguments("--disable-gpu");
-      DesiredCapabilities chrome = BrowserUtil.chrome();
-      chrome.setCapability(ChromeOptions.CAPABILITY, options);
-
-      browsers.add(chrome);
+        sauceLabsLog.setLevel(Level.SEVERE);
+        selenideRemoteLog.setLevel(Level.WARNING);
     }
 
-    if (testBrowsers.contains(TestApp.Browser.FIREFOX)) {
-      browsers.add(BrowserUtil.firefox());
+    private static final int SERVER_PORT =
+            Integer.parseInt(
+                    System.getProperty(TestApp.Properties.SERVER_PORT, TestApp.Defaults.SERVER_PORT));
+    private static final String LOCAL_URL =
+            String.format("http://host.testcontainers.internal:%d", SERVER_PORT);
+    private static final String REPORT_DIRECTORY =
+            System.getProperty(TestApp.Properties.REPORT_DIR, TestApp.Defaults.REPORT_DIR);
+    protected static final String BASE_URL =
+            System.getProperty(TestApp.Properties.TEST_URL, LOCAL_URL);
+
+    private static final String BUILD_NAME =
+            System.getProperty(TestApp.Properties.BUILD_NAME, TestApp.Defaults.BUILD_NAME);
+
+    private static String testName;
+
+    @Rule
+    public final TestRule watchman =
+            new TestWatcher() {
+                @Override
+                protected void starting(Description description) {
+                    super.starting(description);
+                    testName = setTestNameFromTestDescription(description);
+                    log.info(String.format("Starting build '%s'. Test: '%s", BUILD_NAME, testName));
+                }
+
+                @Override
+                protected void succeeded(Description description) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter description should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)